lib/command.js

```diff
@@ -194,6 +200,11 @@ module.exports = function (yargs, usage, validation) {
       if (maybe.variadic) argv[maybe.cmd] = argv._.splice(0)
       else argv[maybe.cmd] = argv._.shift()
       postProcessPositional(yargs, argv, maybe.cmd)
```

Comment:
I agree with @ssonal about the order of the logic; otherwise we'd be doing double duty calling coercion functions for both the normal and expanded form of the argument.

I also agree with @bcoe that we should include another test to make sure coercion is applied to both forms.